src/domain/child/modal/ChildFormModal.tsx

```diff
@@ -156,7 +156,7 @@ const ChildFormModal: React.FunctionComponent<ChildFormModalProps> = ({
                 </div>
 
                 <EnhancedInputField
-                  name="relationship"
+                  name="relationship.type"
```

Comment:
for the MVP we should keep it as string, if you make this change you gotta change the struc of the reducer data also

Reply:
also typing

Reply:
Which reducer? 
Now I don't understand what you mean - without this change the backend gives us an error when we try to submit data.